pkg/service/record/service.go

```diff
@@ -1,14 +1,13 @@
 package record
 
 import (
-	"time"
-
 	"go.keploy.io/server/pkg/models"
 	"go.keploy.io/server/pkg/platform"
 	"go.keploy.io/server/pkg/platform/telemetry"
 )
 
 type Recorder interface {
-	CaptureTraffic(path string, proxyPort uint32, appCmd, appContainer, networkName string, dirName string, Delay uint64, buildDelay time.Duration, ports []uint, filters *models.TestFilter, tcDB platform.TestCaseDB, tele *telemetry.Telemetry, passThroughHosts []models.Filters)
-	StartCaptureTraffic(path string, proxyPort uint32, appCmd, appContainer, networkName string, Delay uint64, buildDelay time.Duration, ports []uint, filters *models.TestFilter, enableTele bool, passThroughHosts []models.Filters)
+	StartCaptureTraffic(options models.RecordOptions)
+  	CaptureTraffic(options models.RecordOptions,  dirName string, tcDB platform.TestCaseDB, tele *telemetry.Telemetry)
+	
 }
```

Comment:
instead of using dirName and variables after that. Better to make another struct `RecordingEnvironment` as the reason for making struct was to reduce number of variables passed.

Reply:
Added And Refactored